includes/WcfceMailProcessor.inc

```diff
@@ -416,6 +416,84 @@ class WcfceMailProcessor {
    *   An exception generated by civicrm_api3().
    */
   private function getContactIdByEmail($email) {
+    $res = $this->getContactIdByEmailFromWebform($email);
```

Comment:
Can we rename this to `result` or `data` instead of `res`, will be more readable.